docs/content/features/run_on_aws.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Run on AWS EC2 Instances"
+draft: false
+description: "Run Infection Monkey on AWS EC2 instances"
+tags: ["aws", "ec2", "run"]
+pre: "<i class='fa-brands fa-aws'></i> "
+---
+
+## Description
+
+The Infection Monkey can run on AWS EC2 instances using the AWS Systems Manager
+(SSM) Agent. The SSM Agent is software installed on each EC2 instance that
+allows for updates, management, and configuration. It receives requests from
+AWS Systems Manager and executes them on the instance, using the Amazon Message
+Delivery Service to report back status and execution details.
+
+This feature lets you execute the Infection Monkey Agent on selected EC2
```

Comment:
"selected"? On what basis? Or do you mean selected by the user?

Reply:
Selected by the user